src/AcceptanceTests/When_using_Aes_with_custom.cs

```diff
@@ -42,11 +42,11 @@ public async Task Should_receive_decrypted_message()
                 Assert.That(context.Secret, Is.EqualTo("betcha can't guess my secret"));
                 Assert.That(context.SubPropertySecret, Is.EqualTo("My sub secret"));
             });
-            CollectionAssert.AreEquivalent(new List<string>
+            Assert.That(context.CreditCards, Is.EquivalentTo(new List<string>
```

Comment:
These could probably simplified into collection expressions but again out of scope